src/features/people/PersonList/styled.js

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 208px);
+  grid-gap: 16px;
```

Comment:
In Figma project grid-gap is 16px on mobiles, in larger screens it is 24px.

Reply:
changed the grid-gap to 24px on bigger screens and to 16px on mobiles.